CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Fix `error_reporting()` not being evaluated when handling errors when the `error_types` option is not set (#1196)
```

Comment:
I think we should clarify this better, with something like this:
```suggestion
- Changes behaviour of `error_types` option when not set: before it defaulted to `error_reporting()` statically at SDK initialization; now it will be evaluated each time during error handling to allow silencing error temporary dynamically (#1196)
```

Reply:
Sounds pretty good 👍